### PR TITLE
Fix packaging tests not failing, when they should

### LINKED ISSUE #1266: CI packaging tests not failing, when they should
1) Update to this commit, [link](https://github.com/CesiumGS/cesium-unreal/commit/a319ea41dca7e70bed0cda97b188dd8bce1f8525)
2) Build the cesium-unreal plugin
3) Open cesium-unreal-samples, default map (01_Cesium_World)
4) Package it for Windows

You'll see this failure
image

The fix is trivial, just remove the link break here, (line 88)
image

The real issue is that our CI packaging tests didn't report this error. 

Here is a related run, [link](https://github.com/CesiumGS/cesium-unreal/actions/runs/6608645360)

We should make our packaging test fail for this commit, and for any other related problems.

### PR DESCRIPTION
**Pull request by csciguy8**
Closes #1266 

CI packaging tests now detect errors in `Engine.ini`, as demonstrated in [this run](https://github.com/CesiumGS/cesium-unreal/actions/runs/6734540006).

The tradeoff is now we run with `-cook` instead `-cookonthefly`, which makes the packaging tests run much slower than before. It's probably fair to test cooking our plugin assets, but this does make our slow CI builds even slower.

Here is another run that shows packaging tests times before these changes (19-24 mins), [here](https://github.com/CesiumGS/cesium-unreal/actions/runs/6725957368)

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @csciguy8!

* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] [Cesium for Unreal Samples](https://github.com/CesiumGS/cesium-unreal-samples) works.

File: .github/workflows/testPackageOnWindows.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
...
+          ./RunUAT.bat -ScriptsForProject="$env:TESTS_PROJECT_ROOT/TestsProject.uproject" Turnkey -command=VerifySdk -platform=Win64 -UpdateIfNeeded -EditorIO -EditorIOPort=54342  -project="$env:TESTS_PROJECT_ROOT/TestsProject.uproject" BuildCookRun -nop4 -utf8output -nocompileeditor -skipbuildeditor -$env:UNREAL_COOK_PARAM  -project="$env:TESTS_PROJECT_ROOT/TestsProject.uproject"  -unrealexe="${{ inputs.unreal-binaries-path }}/UnrealEditor-Cmd.exe" -platform=Win64 -ddc=InstalledDerivedDataBackendGraph -installed -stage -archive -package -build -compressed -iostore -pak -prereqs -archivedirectory="$env:GITHUB_WORKSPACE/TestPackaging" -clientconfig=Shipping -nodebuginfo -nocompile -nocompileuat
```
Comment (csciguy8) [inline .github/workflows/testPackageOnWindows.yml:83]: Updated packaging script to what you'd see when packaging via the Unreal Editor, v5.3.1

File: TestsProject/Config/DefaultEngine.ini [hunk 2]
```diff
@@ -98,3 +98,6 @@ ManualIPAddress=
 +CollisionChannelRedirects=(OldName="VehicleMovement",NewName="Vehicle")
 +CollisionChannelRedirects=(OldName="PawnMovement",NewName="Pawn")
 
+[/Script/UnrealEd.CookerSettings]
```
Comment (csciguy8) [inline TestsProject/Config/DefaultEngine.ini:101]: When packaging from the editor, and checking "Enable cooking on the fly", this is the resultant change. Although from what I see when packaging from the command line, it is not cooking on the fly. It's doing the cook normally

**Comment by csciguy8** [comment]
Not really sure this is a good candidate for a merge, quite yet.

Latest CI times for a successful packaging job takes from 52-148 mins. This is up from 19-24 mins.

**Comment by csciguy8** [comment]
Closing this PR for now. 

My last comment suggests we shouldn't merge this due to the increased CI time, and I still think this.

It could be that a proper solution does what this PR does, but additionally solves some of the CI time issues.

**Pull request closed without merging**